src/ui-utils/clients.ts

```diff
@@ -23,27 +29,35 @@ export function isLaunchpadClient(
   return client instanceof LaunchpadClient;
 }
 
+export function isStargateSigningClient(
+  signingClient: LaunchpadSigningClient | StargateSigningClient | null,
+): signingClient is StargateSigningClient {
+  return signingClient instanceof StargateSigningClient;
+}
+
+export function isLaunchpadSigningClient(
+  signingClient: LaunchpadClient | StargateClient | null,
+): signingClient is LaunchpadSigningClient {
+  return signingClient instanceof LaunchpadSigningClient;
+}
+
 export function generateMnemonic(): string {
   return Bip39.encode(Random.getBytes(16)).toString();
 }
 
-export function loadOrCreateMnemonic(): string {
+export function loadOrCreateMnemonic(mnemonic?: string): string {
   const key = "burner-wallet";
-  const loaded = localStorage.getItem(key);
```

Comment:
If the mnemonic is never loaded, this user gets a new one every time the page is reloaded. This seems to be annoying for testing. Do you have a specific reason from removing the load functionality?

Reply:
It was a naive way of avoiding that the user got stuck with an account without funds. I just pushed some changes that keep the account stored, but if the user enters another mnemonic it replaces it.

Reply:
> but if the user enters another mnemonic it replaces it

Yeah that's good